paddle/fluid/distributed/fleet_executor/carrier.cc

```diff
@@ -46,7 +48,8 @@ void Carrier::Init(
     const std::unordered_map<int64_t, int64_t>& interceptor_id_to_rank,
     const std::unordered_map<int64_t, TaskNode*>& interceptor_id_to_node,
     const framework::ProgramDesc& program, framework::Scope* scope,
-    int64_t num_micro_batches, const platform::Place& place) {
+    int64_t num_micro_batches, const platform::Place& place,
+    const std::vector<std::string>& force_root_scope_var_names) {
```

Comment:
Does the last parameter only be used Inference?

Reply:
done updated, add infer key word in the param's name